astroid/brain/brain_typing.py

```diff
@@ -107,13 +107,11 @@ def infer_typedDict(  # pylint: disable=invalid-name
     """Replace TypedDict FunctionDef with ClassDef."""
     class_def = nodes.ClassDef(
         name="TypedDict",
-        doc=node.doc,
         lineno=node.lineno,
```

Comment:
Why is it necessary to remove the doc?

Reply:
Tbh I don't really know why I added it in the first place. AFAIK it isn't necessary.

Reply:
Ok let's remove it so. :+1: